Pangolin/UnitTest/Framework/ConfigurationTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using EnderPi.Framework.DataAccess;
+using NUnit.Framework;
+
+namespace UnitTest.Framework
+{
+    public class ConfigurationTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+
+        [Test]
+        public void TestGlobalSettings()
+        {
+            string settingName = Guid.NewGuid().ToString();
+            string settingValue = Guid.NewGuid().ToString();
+            string settingValue2 = Guid.NewGuid().ToString();
```

Comment:
Use some hardcoded guids!!!

If/whenever you get around to building a database integrity/first aid, you can remove these values from your tables should they exist. If you have a unit test fail after inserting mock data, you can use the tool to cleanup and remove unit test data based on these hardcoded guids.